ui/src/Grid/Message.elm

```diff
@@ -0,0 +1,7 @@
+module Grid.Message exposing (..)
+
+import Grid.Player exposing (..)
+
+type Msg = PlayerMoved PlayerId Position Orientation
+         | PlayerDied PlayerId
+         | NoOp
```

Comment:
Why not share that message structure between the server and client then? Same problem of it not being a proper subset of all messages?